crates/sui/src/client_commands.rs

```diff
@@ -197,11 +199,12 @@ pub enum SuiClientCommands {
     },
 
     /// Obtain all gas objects owned by the address.
+    /// An address' alias can be used instead of the address.
     #[clap(name = "gas")]
     Gas {
-        /// Address owning the objects
+        /// Address (or its alias) owning the objects
         #[clap(name = "owner_address")]
-        address: Option<SuiAddress>,
+        address: Option<String>,
```

Comment:
More of a high-level thought than a full review on this--would using an `enum` type that cases out on `SuiAddress` versus another `Alias` type help us out here? That way, the `alias_or_address` code logic could convert to the desired type early parsing / input stage and our code internals reflect these as input types. And, if we need internal conversion, maybe implementing the `From` trait is the answer?

Reply:
Agreed, I shall look into that!